### PR TITLE
Fix map frame transformation

### LINKED ISSUE #26: map_to_camera translation won't work well
When running OpenVSLAM-ROS node with Gazebo, it was detected that robot's system coordinate (OXY plane for robot_frame-odom) is orthogonal to map system coordinate (OXY plane for map) as depicted below:
issue_01_r

For the camera frame z-axis is always looks in the same direction as the camera while map plane is rotated to have OXY plane to be parallel to map. So, the following translation is being applied:
P10616-160721_r

It looks like the following place in the `openvslam_ros.cc` is responsible for that:
```
    Eigen::Matrix3d cv_to_ros;
    cv_to_ros << 0, 0, 1,
        -1, 0, 0,
        0, -1, 0;

    // Transform from CV coordinate system to ROS coordinate system on camera coordinates
    map_to_camera_affine.prerotate(cv_to_ros).rotate(cv_to_ros.transpose());
```

However, it is strange why the transform is being applied twice there? If we are applying some rotation, I think it will be enough just to multiply by rotation matrix once, e.g. as made [here](https://github.com/OpenVSLAM-Community/openvslam_ros/blob/ros2/src/openvslam_ros.cc#L29). Also, experiments shown that after the above part of code `map_to_camera_affine`  hasn't changed (related TF will be identical) if apply `prerotate(cv_to_ros).rotate(cv_to_ros.transpose())` to it.

So, I suppose that here should be something like this instead:
```
-    map_to_camera_affine.prerotate(cv_to_ros).rotate(cv_to_ros.transpose());
+    map_to_camera_affine.rotate(cv_to_ros);
```
or:
```
    Eigen::Matrix3d cv_to_ros;
    cv_to_ros << 0, 0, 1,
        -1, 0, 0,
        0, -1, 0;
    Eigen::Affine3d map_to_camera_affine(trans * rot * cv_to_ros);
```
without further `rotate()` routine.

Both cases are works well for me: map and camera's planes became to be correctly aligned.
Any thoughts on that?

### PR DESCRIPTION
**Pull request by ymd-stella**
Related to #26.

### CODE REVIEW

**Comment by ymd-stella** [comment]
@AlexeyMerzlyakov Does it behave as intended?

File: src/openvslam_ros.cc [hunk 1]
```diff
@@ -32,8 +32,8 @@ void system::publish_pose(const Eigen::Matrix4d& cam_pose_wc, const rclcpp::Time
         -1, 0, 0,
         0, -1, 0;
 
-    // Transform from CV coordinate system to ROS coordinate system on camera coordinates
-    map_to_camera_affine.prerotate(cv_to_ros).rotate(cv_to_ros.transpose());
+    // Transform map frame from CV coordinate system to ROS coordinate system
+    map_to_camera_affine.prerotate(cv_to_ros);
```
Comment (AlexeyMerzlyakov) [inline src/openvslam_ros.cc]: As I understand, this is transformation matrix from ROS map coordinate system to OpenCV map coordinate system. Shouldn't it be named as `ros_to_cv`?

**Review by AlexeyMerzlyakov — APPROVED** [review]
Works fine for me. Thanks!

File: src/openvslam_ros.cc (hunk 1 above)
Comment (ymd-stella) [reply src/openvslam_ros.cc]: Thank you very much. I renamed it.

**Pull request merged**